python/triton/experimental/gluon/language/_core.py

```diff
@@ -501,6 +501,11 @@ def warp_specialize(default_args, default_partition, worker_args, worker_partiti
                                      worker_num_regs, _generator)
 
 
+@builtin
+def get_num_warps(_semantic=None, _generator=None):
+    return _semantic.get_num_warps(_generator)
```

Comment:
Should probably be `num_warps` to match the existing `num_programs`.
```suggestion
def num_warps(_semantic=None, _generator=None):
    """
    Returns the number of warps that execute the current context, including in warp-specialized regions.
    """
    return _semantic.get_num_warps(_generator)
```